projects/core/styles/mixins/mixins.less

```diff
@@ -53,7 +53,31 @@
     line-height: inherit;
 }
 
-.clearinput() {
+.autofill(@mode:default) {
+    -webkit-text-fill-color: currentColor; // for Safari
```

Comment:
I don't think it's autofill related, probably could be moved back to clearinput

Reply:
fixed